### ISSUE TITLE
[BLOCKED] Abstract not showing in index view and needs show/hide behavior

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/SearchWorks#1483]
The view code for the search results tries to put the abstract into the results (assuming that's what we want), but the way we're calling the EDS API, we're not getting the abstract. I believe this is because we're using the EDS `:view` parameter with `brief` (default) instead of `detailed` 

See https://github.com/sul-dlss/SearchWorks/blob/article-search/app/views/article/_index_default.html.erb#L16-L18

From the EDS docs on "view": 
> Specifies the amount of data to return with the response. Title: title only; Brief: Title + Source, Subjects; Detailed: Brief + full abstract

See https://github.com/ebsco/edsapi-ruby/issues/49 and https://github.com/sul-dlss/SearchWorks/pull/1477

**Comment by drh-stanford** [comment]
The new EDS gem supports abstracts in the search results and they look like the following:

![screen shot 2017-07-18 at 12 06 38 pm](https://user-images.githubusercontent.com/1861171/28334949-99c6871c-6bb1-11e7-950e-e5fdbb52c926.png)

@jvine is this as expected?

**Comment by jvine** [comment]
Ah. Yes, as expected, but it makes the show/hide behaviour more urgent. @drh-stanford can we postpone this for now? I'm glad it's available, but I don't think we want to demo it as-is.

**Comment by drh-stanford** [comment]
Blocked by show/hide behavior in https://github.com/sul-dlss/SearchWorks/issues/1445
